usersync/chooser.go

```diff
@@ -147,7 +159,7 @@ func (c standardChooser) evaluate(bidder string, syncersSeen map[string]struct{}
 	}
 	syncersSeen[syncer.Key()] = struct{}{}
 
-	if !syncer.SupportsType(syncTypeFilter.ForBidder(bidder)) {
+	if !syncer.SupportsType(syncTypeFilter.ForBidder(strings.ToLower(bidder))) {
```

Comment:
Can't this line also use `bidderNormalized.String()`? And we don't have include the `strings` package into this file?

Reply:
We cannot use normalised bidder name here and the reason is there is biddersLookup map that is created in `NewSpecificBidderFilter` function with [key of the map being lower cased. ](https://github.com/prebid/prebid-server/blob/88209d7ca5968facbcd42542217fa470d83ab3c0/usersync/bidderfilter.go#L47-L48). This map is used when we call `syncTypeFilter.ForBidder` function. Hope this clarifies.

Reply:
I see, but when I subbed in `bidderNomarlized.String()` for `strings.ToLower(bidder)`, all of the tests still passed. This is more of a nitpick if anything, so I'm happy to approve if you want to stick with the current code.

Reply:
@AlexBVolcy It is because test cases are setup that way. If you setup a test case where you use `suntContent` as bidder name, you will be able to fail your test. In fact I have updated one of the test to reflect this - [c124abb](https://github.com/prebid/prebid-server/pull/3103/commits/c124abb52bfa929451685c5ee5a32569bba32f3f)